src/app/debug/table/table.component.ts

```diff
@@ -18,11 +26,11 @@ import { TabService } from '../../shared/services/tab.service';
 })
 export class TableComponent implements OnInit, OnDestroy {
   DEFAULT_DISPLAY_AMOUNT: number = 10;
-  metadataCount = 0;
-  viewSettings: any = {
+  metadataCount: number = 0;
+  viewSettings: ViewSettings = {
     defaultView: '',
-    views: [],
-    currentView: {},
+    views: {},
+    currentView: {} as View,
```

Comment:
Same case with this, either make currentView be optional/able to be undefined or give it a proper default value